playground/01-pattern-matcher/README.md

```diff
@@ -0,0 +1,21 @@
+## 2. Pattern Matcher
```

Comment:
why 2. ?

Reply:
Because it is copied from an old assignment with many sub-tasks.